src/main/java/svc/security/HashUtil.java

```diff
@@ -0,0 +1,59 @@
+package svc.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HashUtil{
+	@Value("${stlcourts.security.municipalitySalt}")
+	String municipalitySalt;
+	@Value("${stlcourts.security.judgeSalt}")
+	String judgeSalt;
+	@Value("${stlcourts.security.courtSalt}")
+	String courtSalt;
+	
+	public Hashids municipalityHashids;
+	public Hashids judgeHashids;
+	public Hashids courtHashids;
+	
+	private void initializeSalts(){
```

Comment:
Why is this function necessary? The 3 Hashids objects should be able to be created statically, no? We shouldn't have to new them up each time the function is called. Then encode/decode can be static. The whole util should be able to be static. You may to make the salts final or `@Shared` or something